app/Exceptions/AlreadySubscribedException.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
Let's not make many PRs with partial changes, so let's start by deciding where we should have that in production (pros and cons) and then apply on everything we have decided (and tested!) at once.

Reply:
I don't see any disadvantages or problems. on the contrary, the code is more robust because it does what it says.

Reply:
We will probably add those lines, but examples of disadvantages are increased risks of crash in production, and tiny impact on performance (due to additional checks at runtime)...